src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -1250,6 +1267,117 @@ void AssimpImporterTest::animationMerge() {
     CORRADE_COMPARE(scaling2.interpolation(), Animation::Interpolation::Linear);
 }
 
+/* The checks are identical to animation support so re-use that. */
+bool supportsSkinning(const Containers::StringView fileName) {
+    return supportsAnimation(fileName);
+}
+
+void AssimpImporterTest::skin() {
+    auto&& data = ExportedFileData[testCaseInstanceId()];
+    setTestCaseDescription(data.name);
+
+    if(!supportsSkinning(data.suffix))
+        CORRADE_SKIP("Skin data for this file type is not supported with the current version of Assimp");
+
+    /* Skinned mesh imported into Blender and then exported */
+
+    // TODO attribution:
+    // https://github.com/KhronosGroup/glTF-Tutorials/blob/master/gltfTutorial/gltfTutorial_019_SimpleSkin.md
```

Comment:
Where would I best put attribution here? The tutorials are licensed under [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/) so anything allowed as long as we credit the author and add a link to the license. I'm afraid recreating a skinned mesh with Blender is a bit out of my depth.

Reply:
Is that where the `skin.gltf` is from? Looks rather different, did you re-export with Blender? The test files from TinyGltfImporter could not be reused?

In any case, attribution should go [here](https://github.com/mosra/magnum-plugins/blob/faa16d460d21dc8eb3682aec4335954db5dca758/src/MagnumPlugins/AssimpImporter/AssimpImporter.h#L127-L131), something like

```patch
diff --git a/src/MagnumPlugins/AssimpImporter/AssimpImporter.h b/src/MagnumPlugins/AssimpImporter/AssimpImporter.h
index 0da7c4c3..2d03fa5c 100644
--- a/src/MagnumPlugins/AssimpImporter/AssimpImporter.h
+++ b/src/MagnumPlugins/AssimpImporter/AssimpImporter.h
@@ -128,7 +128,11 @@ This plugin provides `3dsImporter`, `Ac3dImporter`, `BlenderImporter`,
     licensed under @m_class{m-label m-success} **BSD 3-clause**
     ([license text](http://assimp.org/index.php/license),
     [choosealicense.com](https://choosealicense.com/licenses/bsd-3-clause/)).
-    It requires attribution for public use.
+    It requires attribution for public use. The tests use a file from the
+    [A Simple Skin](https://github.com/KhronosGroup/glTF-Tutorials/blob/master/gltfTutorial/gltfTutorial_019_SimpleSkin.md)
+    glTF tutorial, licensed under @m_class{m-label m-success} **CC-BY 4.0**
+    ([license text](https://github.com/KhronosGroup/glTF-Tutorials/blob/master/LICENSE),
+    [choosealicense.com](https://choosealicense.com/licenses/cc-by-4.0/)).
 
 @section Trade-AssimpImporter-usage Usage

```

Reply:
Yeah, it's re-exported from Blender, mainly because I needed an extra un-skinned mesh in the scene. The *original* TinyGltfImporter skin test files can't be used with Assimp because they have no meshes. Assimp only exposes bones through meshes, so no meshes = no bones. Main reason I imported and re-exported that tutorial file is that it has accompanying joint weights, making it fairly simple to test both skins and vertex attributes with the same file.

Reply:
@mosra turns out the original file is from the [glTF 2 sample repo](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/SimpleSkin), licensed under [CC0](https://creativecommons.org/choose/zero/). Do I still mention it in that case? It's not used verbatim, if that makes any difference.

Reply:
I don't think that's needed, no.